src/app/ReadClient.h

```diff
@@ -183,6 +183,15 @@ class ReadClient : public Messaging::ExchangeDelegate
             aEncodedDataVersionList = false;
             return CHIP_NO_ERROR;
         }
+
+        /*
+         * Get highest received event number.
```

Comment:
Needs to clearly document semantics, which are that if you don't have one you clear outparam and return CHIP_NO_ERROR, and that returning error will fail the entire read client.

Reply:
Done, thanks